versions.tf

```diff
@@ -1,8 +1,9 @@
 terraform {
-  required_version = ">= 0.12.0, < 0.14.0"
+  required_version = ">= 0.13.0, < 0.14.0"
```

Comment:
I know @danjbh said to bump this, but not sure if we're using 0.13 constructs. Our test harness for 0.13 is not live yet.

Reply:
...meaning we cannot run `/test all` on this yet if this is true.

Reply:
Keeping it at 0.12 for now